backend/src/api/organization/access-code.controller.ts

```diff
@@ -0,0 +1,92 @@
+import {
+  Body,
+  Controller,
+  Delete,
+  Get,
+  Param,
+  Patch,
+  Post,
+} from '@nestjs/common';
+import { ApiBody, ApiParam } from '@nestjs/swagger';
+import { CreateAccessCodeUseCase } from 'src/usecases/access-code/create-access-code.usecase';
+import { DeleteAccessCodeUseCase } from 'src/usecases/access-code/delete-access-code.usecase';
+import { GetAccessCodeUseCase } from 'src/usecases/access-code/get-access-code.usecase';
+import { GetAllAccessCodeUseCase } from 'src/usecases/access-code/get-all-access-codes.usecase';
+import { UpdateAccessCodeUseCase } from 'src/usecases/access-code/update-access-code.usecase';
+import { CreateAccessCodeDto } from './dto/create-access-code.dto';
+import { UpdateAccessCodeDto } from './dto/update-access-code.dto';
+import { AccessCodePresenter } from './presenters/access-code.presenter';
+
+// @Roles(Role.ADMIN)
+// @UseGuards(WebJwtAuthGuard)
+// @UsePipes(new UuidValidationPipe())
+@Controller('organization/access-code')
+export class AccessCodeController {
+  constructor(
+    private readonly createAccessCodeUseCase: CreateAccessCodeUseCase,
+    private readonly updateAccessCodeUseCase: UpdateAccessCodeUseCase,
+    private readonly deleteAccessCodeUseCase: DeleteAccessCodeUseCase,
+    private readonly findAccessCodeUseCase: GetAccessCodeUseCase,
+    private readonly findAllAccessCodeUseCase: GetAllAccessCodeUseCase,
+  ) {}
+
+  @Get('access-code')
+  async getAll(): Promise<AccessCodePresenter[]> {
+    const accessCodes = await this.findAllAccessCodeUseCase.execute({
+      organizationId: '3631315f-02f1-42c9-a418-8bff2e15fb2d', // TODO: replace with organization from @User request
+    });
+
+    return accessCodes.map((accessCode) => new AccessCodePresenter(accessCode));
+  }
+
+  @ApiParam({ name: 'accessCodeId', type: 'string' })
```

Comment:
Here we should have the validation uuid pipe

Reply:
Am adaugat inclusiv in OrganizationStructure (BDR), o sa fie in celalalt PR.